app/controllers/admin/reports/engaged_schools_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Admin
+  module Reports
+    class EngagedSchoolsController < AdminController
+      def index
+        respond_to do |format|
+          format.html do
+            @engaged_schools = ::Schools::EngagedSchoolService.list_engaged_schools
```

Comment:
`` @engaged_schools = ::Schools::EngagedSchoolService.list_engaged_schools``
 is done twice in index and could just be done once at the top of the method?